tests/test_forms.py

```diff
@@ -112,6 +112,11 @@ def test_clean(self, form_data, num_expected_missing_credentials_errors, side_ef
     def test_requires_connection_test(self):
         assert JiraAuthenticationForm()._requires_connection_test()
 
+    def test_get_connection(self):
+        jira_authentication_form = JiraAuthenticationForm()
+        with pytest.raises(NotImplementedError):
```

Comment:
You can simply mark the method stub with `# pragma: no cover` and remove this test. Not only is raising a `NotImplementedError` the common way to define a method stub, but this test doesn't really test anything useful either. It basically just tests that Python actually raises an exception when one writes `raise NotImplementedError()`.